packages/twenty-server/src/engine/metadata-modules/object-metadata/hooks/before-update-one-object.hook.ts

```diff
@@ -100,6 +101,8 @@ export class BeforeUpdateOneObject<T extends UpdateObjectInput>
       }
     }
 
+    formatObjectMetadataInput(instance.update);
```

Comment:
object names can actually not be updated (cf `checkIfFieldIsEditable`) but it's important we got the formatted string here because it is compared to the previous name value in `checkIfFieldIsEditable` to check if there is an update or nos